src/files/coverageparser.ts

```diff
@@ -200,11 +200,14 @@ export class CoverageParser {
         }
     }
 
-    private lcovExtract(lcovFile: string) {
+    private lcovExtract(filename: string, lcovFile: string) {
         return new Promise<Map<string, Section>>((resolve, reject) => {
             try {
                 source(lcovFile, async (err, data) => {
-                    if (err) { return reject(err); }
+                    if (err) {
```

Comment:
Removed the reject since the calling code doesn't have a try catch nor a then for the promise so it fails and no coverage is shown. I think this new change was the intended behavior to return empty coverage for invalid files

Reply:
On another note the exact same problem will happen for the other file types as well, I wanted to stick to the lcov for now since I don't really know what I am doing in your project ;)

Reply:
Yah exposing these errors to the developer via the output channels is a better call, as you could still use the extension with 1 out of 3 coverage files in error for example.